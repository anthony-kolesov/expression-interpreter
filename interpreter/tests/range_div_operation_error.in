out {1, 3} / 4
