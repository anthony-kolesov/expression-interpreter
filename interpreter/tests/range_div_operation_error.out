ERROR:1:Cannot perform arithmetic operation on vector values.
